src/transformers/models/ctrl/modeling_ctrl.py

```diff
@@ -371,6 +364,23 @@ def forward(
         output_hidden_states=None,
         return_dict=None,
     ):
+        r"""
+        Returns:
+
+        Example:
+
+        ```python
+        >>> from transformers import CTRLTokenizer, CTRLModel
+        >>> import torch
+
+        >>> tokenizer = CTRLTokenizer.from_pretrained("sshleifer/tiny-ctrl")
+        >>> model = CTRLModel.from_pretrained("sshleifer/tiny-ctrl")
+
+        >>> inputs = tokenizer("Opinion my dog is cute", return_tensors="pt")
+        >>> outputs = model(**inputs)
+
+        >>> last_hidden_states = outputs.last_hidden_state
```

Comment:
Since we overwrite the sample here, we can actually provide the expected value, and let the doctest to test it.